ios/BlurView.m

```diff
@@ -37,12 +37,13 @@ - (void)setBlurType:(NSString *)blurType
     } else {
         self.blurEffect = [UIBlurEffect effectWithStyle:UIBlurEffectStyleDark];
```

Comment:
I agree with you with the [BlurAmount +effectWithStyle] instead of UIBlurEffect. 
But when BlurView is dealloc by ARC on iOS8, it still crash [image of crash stack](https://raw.githubusercontent.com/haywoodfu/Notes/master/Images/stacktrace-20170413.png)  . I can't fix it with subclassing,  any tips for this  crash? thanks~